### PR TITLE
list website workgroup members

### PR DESCRIPTION
**Pull request by tomerd**
motivation: share information about the newly formed workgroup

changes: add a section to the website workgroup page with names of members

### CODE REVIEW

File: website-workgroup/index.md [hunk 1]
```diff
@@ -14,6 +14,20 @@ The Swift website workgroup is a steering team that helps guide the evolution on
 
 Analogous to the [Core Team](/community#core-team) for Swift, the workgroup is responsible for establishing the processes and standards by which changes to the Swift.org website are proposed and eventually integrated.
 
+The current website work group consists of the following people:
+
+* Alexander Sandberg ([@alexandersandberg](https://github.com/alexandersandberg))
+* Dave Verwer ([@daveverwer](https://github.com/daveverwer))
+* James Dempsey ([@James_Dempsey](https://github.com/?))
```
Comment (shahmishal) [inline website-workgroup/index.md:21]:
```suggestion
* James Dempsey ([@James_Dempsey](https://github.com/dempseyatgithub))
```

**Review by shahmishal — APPROVED** [review]
provided missing GitHub username for @dempseyatgithub

**Review by daveverwer — APPROVED** [review]

File: website-workgroup/index.md (hunk 1 above)
Comment (alexandersandberg) [reply website-workgroup/index.md:21]:
```suggestion
* James Dempsey ([@dempseyatgithub](https://github.com/dempseyatgithub))
```

Fixing the @ as well 🙂

**Comment by shahmishal** [comment]
@swift-ci test

**Review by 0xTim — APPROVED** [review]
LGTM

**Comment by krstnfx** [comment]
Looks good

**Pull request merged**